api/src/sockets/kots/DeploySocket.ts

```diff
@@ -131,8 +131,16 @@ export class KotsDeploySocketService {
                 kubectl_version: kotsAppSpec ? kotsAppSpec.kubectlVersion : "",
                 namespace: desiredNamespace,
                 manifests: b.toString("base64"),
+                previous_manifests: "",
               };
 
+              const previousSequence = await this.kotsAppStore.getPreviouslyDeployedSequence(app.id, clusterSocketHistory.clusterId, deployedAppSequence!);
+              if (previousSequence !== undefined) {
+                const previousRendered = await app.render(''+(previousSequence-1), `overlays/downstreams/${cluster.title}`);
```

Comment:
```suggestion
                const previousRendered = await app.render(''+previousSequence, `overlays/downstreams/${cluster.title}`);
```
Since this should be the previous sequence, not the sequence after the previous sequence